src/Command/CreateManifestCommand.php

```diff
@@ -0,0 +1,87 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\AppBundle\Command;
+
+use DOMDocument;
+use DOMXPath;
+use Shopware\AppBundle\Service\AnnotationReaderService;
+use Shopware\AppBundle\Service\ManifestCreationService;
+use SimpleXMLElement;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+class CreateManifestCommand extends Command
+{
+    private const MANIFEST_DESTINATION = '../development/custom/apps/AppTemplate/manifest.xml';
```

Comment:
Is this just debug code? We should make this path configurable through our config. A default should be something like

```
%kernel.project_dir%/build/%app_env%
```

Reply:
Oh yes this is just to make it easy for me :smile:   
This will be configurable in the final mr.

Reply:
This is now configurable in the `shopware_app.yaml` and must be configured in order to create the manifest.